services/ai_client.py

```
@@ -0,0 +1,103 @@
+        }
+        
+        try:
+            response = requests.post(
+                self.base_url,
+                headers=headers,
+                json=payload,
+                timeout=self.timeout_ms / 1000.0  # Convert to seconds
+            )
+            
+            if response.status_code != 200:
+                raise AIClientError(f"API request failed with status {response.status_code}: {response.text}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)